libraries/rush-lib/src/api/Subspace.ts

```diff
@@ -321,4 +325,71 @@ export class Subspace {
   public _addProject(project: RushConfigurationProject): void {
     this._projects.push(project);
   }
+
+  /**
+   * Returns hash value of injected dependencies in related package.json.
+   * @beta
+   */
+  public getPackageJsonInjectedDependenciesHash(): string | undefined {
```

Comment:
Because this is a somewhat heavy operation, can you make this a normal function instead of a getter?

Reply:
Isn't it already a normal function?

Reply:
Maybe I misread this earlier when I thought this was a getter.